src/build/index.cjs

```diff
@@ -0,0 +1,6 @@
+const buildThemedComponents = async (...args) => {
+  const esmAPI = await import('./index.js');
+  return esmAPI.buildThemedComponents(...args);
+};
```

Comment:
Thanks to webdriverio for this idea how to support compatibility without code duplication:

https://github.com/webdriverio/webdriverio/blob/63ac0e07bef1ac9e9075bbef24bd9c9340085bdc/packages/webdriverio/src/index.cts#L114-L120